support-frontend/conf/CODE.public.conf

```diff
@@ -17,7 +17,7 @@ paymentApi.url="https://payment.code.dev-guardianapis.com"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' https://ccpa-pm.sp-prod.net https://cdn.privacy-mgmt.com https://gdpr-tcfv2.sp-prod.net https://ccpa-service.sp-prod.net https://ccpa-notice.sp-prod.net https://sourcepoint.theguardian.com https://ccpa.sp-prod.net https://services.postcodeanywhere.co.uk https://stripe-intent-code.support.guardianapis.com www.paypalobjects.com t.paypal.com/ members-data-api.code.dev-theguardian.com metric-push-api-code.support.guardianapis.com www.paypal.com www.sandbox.paypal.com js.stripe.com https://payments-sandbox.amazon.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' q.stripe.com payment.code.dev-guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals-code.support.guardianapis.com/referral-code"
```

Comment:
hard to see from diff, but this just removed `https://contribution-reminders-code.support.guardianapis.com/` from the list